sdks/go/pkg/beam/transforms/xlang/external.go

```diff
@@ -0,0 +1,129 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package xlang contains data structures required for python external transforms in a multilanguage pipeline.
```

Comment:
I recommend having nothing in the transforms/xlang package, and put this into the python directory instead. It's all about python, and invoking python.

I also recommend simply putting the python utilities into a directory parallel to dataframe instead of trying to impose a hierarchy.

eg.
```
transforms/
  xlang/
    python/
    dataframe/
```

This also lets you avoid having "xlang.Python..." among other prefixes everywhere and instead have python.Callable at the other call sites.

Reply:
this looks clean. thanks!